### PR TITLE
Add contract details doc

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

- Edit contract md file to include updated contract doc

image

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1739

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by 0oM4R — APPROVED** [review]

File: packages/playground/public/info/contracts_list.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 title: Contracts
 ---
 
-You can find detailed information about your [node/gateway contracts](https://manual.grid.tf/tfchain/tfchain.html)
+For more details about Contract Types, Billing Cycle & Grace Period, check [Contract Documentation](https://manual.grid.tf/tfchain/tfchain.html)
+<br />
+
+To explore further contract details, check [Node Contract Documentation](https://manual.grid.tf/playground/contracts_list.html)
+<br />
```
Comment (amiraabouhadid) [inline packages/playground/public/info/contracts_list.md:9]: can we remove all the extra empty space at the bottom?

File: packages/playground/public/info/contracts_list.md [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
 
 - Contract State: Created, GracePeriod (and also know how many tokens are needed to fund contracts in graceperiod to resume your workloads), and the expiration date of these contracts
 - The Billing Rate: How much the contract costs you per hour
```
Comment (amiraabouhadid) [inline packages/playground/public/info/contracts_list.md:12]: Can be just : "The Billing Rate (i.e contract cost per hour)"

File: packages/playground/public/info/contracts_list.md [hunk 3]
```diff
@@ -2,12 +2,13 @@
...
 - Solution type and name if the contract creator provided them
```
Comment (amiraabouhadid) [inline packages/playground/public/info/contracts_list.md:13]: "Solution type and name (if provided by contract creator)"

File: packages/playground/public/info/contracts_list.md [hunk 4]
```diff
@@ -2,12 +2,13 @@
...
 - Full details about the contracts as well in JSON format
```
Comment (amiraabouhadid) [inline packages/playground/public/info/contracts_list.md:14]: "Full contracts details in JSON format"

File: packages/playground/public/info/contracts_list.md [hunk 5]
```diff
@@ -2,12 +2,13 @@
...
 ---
 
-You can find detailed information about your [node/gateway contracts](https://manual.grid.tf/tfchain/tfchain.html)
+For more details about Contract Types, Billing Cycle & Grace Period, check [Contract Documentation](https://manual.grid.tf/tfchain/tfchain.html)
+<br />
+
+To explore further contract details, check [Node Contract Documentation](https://manual.grid.tf/playground/contracts_list.html)
+<br />
 
 - Contract State: Created, GracePeriod (and also know how many tokens are needed to fund contracts in graceperiod to resume your workloads), and the expiration date of these contracts
```
Comment (amiraabouhadid) [inline packages/playground/public/info/contracts_list.md:11]: "Contract State provides you with the following information:
   - an indicator of whether the contract is created or in the grace period
   - the number of tokens needed to fund contracts in the grace period to resume the workload
   - contract expiration date"

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**